Script/AtomicNET/AtomicNET/Core/Profiler.cs

```diff
@@ -12,12 +13,16 @@ public partial class Profiler : AObject
         {
 #if ATOMIC_PROFILING
             var profiler = AtomicNET.GetSubsystem<Profiler>();
-            profiler?.BeginBlock(name, file, line, color, (byte)status);
+            if (profiler != null)
```

Comment:
We don't need AtomicNET here, AObject has GetSubsystem on it, the static AtomicNET flavor is only for getting a subsystem outside of a AObject derived instance.  Also, it might be good to locally cache so doesn't look up per block.